src/init.c

```diff
@@ -48,6 +51,7 @@ _PG_fini(void)
 {
 	_process_utility_fini();
 	_planner_fini();
```

Comment:
BTW, is there a reason we don't call *_fini() in strict reverse order that we call *_init() ?